tests/common.go

```diff
@@ -4854,10 +4869,10 @@ func updatePxRuntimeOpts() error {
 }
 
 //StartTorpedoTest starts the logging for torpedo test
-func StartTorpedoTest(testName, testDescription string, tags []string) {
+func StartTorpedoTest(testName, testDescription string, tags map[string]string, testRepoID int) {
```

Comment:
what is the testRepoId? When you invoke this method you pass `testrailID`, so maybe it is better to name it as `testrailID`?